CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## v0.5.x
+
+#### BC breaks
```

Comment:
Consistent with what? As I use 4 everywhere :smile: 
It is personal preference, but I find ### still to big :)